### ISSUE TITLE
End Metals mod not working

### ISSUE THREAD

**Issue by Gummi13** [AllTheMods/ATM-1#91]
i can get end metals with rftool dimensions but only craft them in to cable facade, can't smelt or grind the ores.  Got some end uranium ore from rftool dimlet and it is just for decoration .

**Comment by OrdinatorStouff** [comment]
can you craft two of them together to make a "normal" ore? If not we can expand that functionality to the end metals too

**Comment by OrdinatorStouff** [comment]
oops I thought we were talking ATM3 my bad -- depends on how those scripts and settings are done. Honestly probably would be some options for them to be handled correctly in ATM1

**Comment by Gummi13** [comment]
only thing i can craft is the cable facade nothing else works tryed furnace from 4 driffrent mods and some grinders and pulverizer.

**Comment by BloodWorkXGaming** [comment]
Well, it is intended that no endmetals spawn in the world, besides the vanilla ones.
This help to lower the amount of ore clutter.

**Comment by Gummi13** [comment]
yes but you can get them with dimensions in rftools but no recipes work cant smelt or do any thing with the ores except 1 decoration recipe

**Comment by BloodWorkXGaming** [comment]
soooo, if you are really motivated, do you could make me a list of normal ores corresponding to those names:
`iron_ore
lapis_ore
redstone_ore
coal_ore
gold_ore
diamond_ore
emerald_ore
copper_ore
tin_ore
lead_ore
silver_ore
zinc_ore
nickel_ore
mercury_ore
platinum_ore
antimony_ore
bismuth_ore
aluminum_ore
cadmium_ore
chromium_ore
iridium_ore
magnesium_ore
manganese_ore
osmium_ore
plutonium_ore
rutile_ore
tantalum_ore
titanium_ore
tungsten_ore
uranium_ore
zirconium_ore`

Those are the nether/end ores.
I would add a nether/end ore -> 2 * normal ores recipes if I had the normal ore for each of them.
